src/advanced_inv.cpp

```diff
@@ -336,7 +336,14 @@ void advanced_inventory::print_items( const advanced_inventory_pane &pane, bool
         const item &it = *sitem.items.front();
         const bool selected = active && index == static_cast<int>( i );
 
-        nc_color thiscolor = active ? it.color_in_inventory() : norm;
+        nc_color thiscolor;
+        if( !active ) {
+            thiscolor = norm;
+        } else if( it.is_food_container() && !it.is_craft() ) {
+            thiscolor = it.contents.all_items_top().front()->color_in_inventory();
+        } else {
+            thiscolor = it.color_in_inventory();
```

Comment:
Why not just put this change inside of `it.color_in_inventory()` instead of in a block that sometimes dodges ` thiscolor = it.color_in_inventory()`?

Wouldn't that give the intended effect and also close up the inconsistency of this change affecting the the AIM UI, the `g`et UI, but *not* the `i`nventory UI?

Reply:
i did put it inside color_in_inventory() at first but that had some nasty side effects.
For one color_in_inventory is used in the main inventory screen on a per item basis but in the get and aim windows on just the toplevel. Which means they have some slightly different requirements.

To explain with an example:
Coloring lets say a tupperware box brown because there is a rotten item inside is fine, in the inventory screen where the user can select it and immediatly sees what's inside that caused the colorchange.  But in aim and get, well as soon as there are more than 1 item in the tupperware it won't show it's contents anymore.
Which means we end up with a "rotten" colored container and no information is given to the user as to why.

This is especially bad, since we have containers that can rot too.
Hence why at least for now i decided to just